lib/table.js

```diff
@@ -32,6 +32,9 @@ function Table (name) {
       column.name = name
       column.nullable = options.nullable
       column.primary = options.primary
+      if (Object.prototype.hasOwnProperty.call(options, 'length')) {
```

Comment:
Why are you rebinding with `.call` here?

Reply:
As per the ES Lint rule (https://eslint.org/docs/rules/no-prototype-builtins) , it will throw “Do not access Object.prototype method 'hasOwnProperty' from target object. It is a 'no-prototype-builtins' error.”, if we try to access the property via `if(options.hasProperty(‘length’))`. So I have refactored the code accordingly.

Reply:
If userland code is taking option input from user sources then that's their own choice - I don't think we need to be going to extreme lengths here. The rest of the codebase uses .hasOwnProperty directly so can we stick with that for now?

Reply:
sure, i have changed it